app/src/benchmark/res/values-night/colors.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright 2023 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<resources>
+    <color name="blue40">#FF006780</color>
```

Comment:
I think these color resources should rather come from the `:core:designsystem` module.
`black30` `blue40` `orange40`, etc.

Reply:
I wondered about this. The design system module defines its colors in kotlin. I'm hesitant to duplicate color definitions in xml just so that they can be reused elsewhere as there is no way if enforcing they are synchronized.

I'd much rather the colors for the app Icon be defined independently as their own values. The names should also be more specific;, i.e `appIconColor` as opposed to `blue40` or something. I wouldn't want these colors used anywhere else.

Reply:
Yes, then inlining all these ad-hoc colors would probably be a better alternative, and would prevent exposing them to the rest of the app.